Set up Storybook

T-Dnzt opened the pull request:
## Reason for change

Set up Storybook.

## Changes

* Set up [Storybook](https://storybook.js.org/)
* Minimal changes to the README file to explain how to start Storybook.

## Related tickets

* https://app.asana.com/0/1140170034526545/1205427339227447

## Notes

I have not removed all the examples and documentation generated by Storybook. I thought it might be useful to have examples around as we start creating additional stories. Let me know if you disagree and I can delete them.
 
Feel free to ignore all the `/src/stories/*` files as they were all generated by Storybook.

cwgw reviewed (approved):
Looks good!

> I have not removed all the examples and documentation generated by Storybook. I thought it might be useful to have examples around as we start creating additional stories. Let me know if you disagree and I can delete them.

I think we ultimately won't want to have a`src/stories` directory—we'll want to colocate stories with components—so maybe we should avoid adding it now.

If we want to keep some example files, maybe it would be better to pick one or two and move them into `src/components`? We could move `Button.tsx` and `Button.stories.tsx` into `src/components/Button` and add a comment like "This is an example file generated by Storybook"?

T-Dnzt commented:
@cwgw Sure thing! I've deleted the `src/stories` folder and moved the example button component to `src/components/StorybookButton`. Let's remove it as soon as we have a few of our own stories in place.
